articles/aks/deploy-confidential-containers-default-policy.md

```diff
@@ -313,54 +315,72 @@ For this preview release, we recommend for test and evaluation purposes to eithe
           targetPort: kafka-consumer
     ```
 
-1. Create a kafka namespace by running the following command:
-
-    ```bash
-    kubectl create namespace kafka
-    ```
-
-1. Install the Kafka cluster in the kafka namespace by running the following command:
-
-    ```bash
-    kubectl create -f 'https://strimzi.io/install/latest?namespace=kafka' -n kafka
-    ```
-
-1. Run the following command to apply the `kafka` cluster CR file.
-
-    ```bash
-    kubectl apply -f https://strimzi.io/examples/latest/kafka/kafka-persistent-single.yaml -n kafka
-    ```
+    > [!NOTE]
+    > Update the value for the pod environment variable `SkrClientAKVEndpoint` to match the URL of your Azure Key Vault, excluding the protocol value `https://`. The current value placeholder value is `myKeyVault.vault.azure.net`. 
```

Comment:
The phrase of "pod environmental variable" has been modified to "pod environment variable".
@MGoedtel Thank you for answering.
Actually there is also one part which has the same issue, but I don't think it is appropriate to do the change here:
> You need to set the *environment variables* from the section **Export environmental variables**

I cannot modify "Export environmental variables" here because it is literality the section title from [another article](https://learn.microsoft.com/en-us/azure/aks/workload-identity-deploy-cluster#export-environmental-variables). You may want to modify this later or I can submit another PR later.

This PR already contain more than 30% changes so maybe not in this PR.

Reply:
@JoeyC-Dev - You can leave that one instance as is. I'll open a PR to fix that section and then update the reference here after this merges.

Reply:
@MGoedtel 
Many thanks. 
I believe all necessary changes has been made already. Kindly review the current version.  
Have a nice day.